### PR TITLE
Check existing ml

### PR DESCRIPTION
**Pull request by EdAguilarB**
## Description
Added check buttons to, in case that there is already a model saved in the experiment, ask the user if run the experiment again and overwrite previous results. 
Also corrected a small error in the FI page that caused the page to crash.

## Linked issues
- #127 

## (Optional) Screenshots
image
image
image
image
image
image
image
image

### CODE REVIEW

File: biofefi/components/forms.py [hunk 1]
```diff
@@ -150,13 +152,17 @@ def fi_options_form():
     )
 
     # Scoring function for permutation importance
-    if st.session_state.get(ConfigStateKeys.ProblemType) == ProblemTypes.Regression:
+    if (
+        st.session_state.get(ConfigStateKeys.ProblemType).lower()
+        == ProblemTypes.Regression
+    ):
```
Comment (dcl10) [inline biofefi/components/forms.py:158]:
```suggestion
    if (
        st.session_state.get(ConfigStateKeys.ProblemType, ProblemTypes.Auto).lower()
        == ProblemTypes.Regression
    ):
```

File: biofefi/components/forms.py [hunk 2]
```diff
@@ -150,13 +152,17 @@ def fi_options_form():
...
         scoring_options = [
             "neg_mean_absolute_error",
             "neg_root_mean_squared_error",
         ]
     elif (
-        st.session_state.get(ConfigStateKeys.ProblemType) == ProblemTypes.Classification
+        st.session_state.get(ConfigStateKeys.ProblemType).lower()
+        == ProblemTypes.Classification
```
Comment (dcl10) [inline biofefi/components/forms.py:165]:
```suggestion
        st.session_state.get(ConfigStateKeys.ProblemType, ProblemTypes.Auto).lower()
        == ProblemTypes.Classification
```

File: biofefi/components/forms.py
Comment (dcl10) [inline biofefi/components/forms.py:1]: I think we should keep it simple for now and just ask if the user wants to totally restart if any models exist at all. The way you've done it now would require a new `if` block for every single model we implement.

**Review by dcl10 — CHANGES_REQUESTED** [review]

File: biofefi/components/forms.py
Comment (EdAguilarB) [reply biofefi/components/forms.py:1]: So, just one check and if the user wants to re run, it will re run everything?

Comment (dcl10) [reply biofefi/components/forms.py:1]: Yes

**Comment by EdAguilarB** [comment]
This is how it is displayed:

image

If user chooses to not rerun and clicks Run Training:

image

**Review by dcl10 — APPROVED** [review]

**Review by dcl10 — CHANGES_REQUESTED** [review]

**Review by dcl10 — CHANGES_REQUESTED** [review]

**Review by dcl10 — APPROVED** [review]

**Pull request merged**